Formula/cquery.rb

```diff
@@ -0,0 +1,18 @@
+class Cquery < Formula
+  desc "C/C++ language server"
+  homepage "https://github.com/cquery-project/cquery"
+  # pull from git tag to get submodules
+  url "https://github.com/cquery-project/cquery.git", :tag => "v20180302",
+                                                      :revision => "f3e9e756e182b122bef8826a77047f6ccf5529b6"
+  head "https://github.com/cquery-project/cquery.git"
```

Comment:
No `head` for new formulas

Reply:
Could we keep it? `cquery` is a relatively new project and is seeing rapidly development. Therefore, there is a need for certain users to follow the HEAD.

Reply:
FWIW, I agree wiht @xu-cheng. Having HEAD woul dbe a good idea. Why is there a ban on head for new formulae?

Reply:
Because it tends to break often, and create more work for us.

Reply:
Add a big disclaimer, like: "YOU'RE BUILDING FROM HEAD. IF IT BREAKS, DEAL WITH IT. WE WON'T HELP YOU. (Patches welcome.)"

Reply:
But then patches have to be reviewed, and merged, by someone 😈 
I'd tend to say: if you need to build from head, convince upstream that they should release more often!